polaris/cli.py

```diff
@@ -13,8 +14,8 @@
 @app.command("login")
 def login(
     client_env_file: Annotated[
-        Optional[str], typer.Option(help="Environment file to overwrite the default environment variables")
-    ] = None,
+        str, typer.Option(help="Environment file to overwrite the default environment variables")
+    ] = ".env",
```

Comment:
Since this is a relative path, Pydantic will look for it in `os.getcwd()`. Essentially, where you are invoking Polaris from. You can specify an absolute path, in which case that's where Pydantic will check.

Reply:
Check. My question was on how the `cwd` is set in different cases, but never mind! That's actually not relevant to this PR and I can run some quick tests myself to explore that question!